### PR TITLE
[NEW] add module auth_oauth_multi_token

### PR DESCRIPTION
**Pull request by fmdl**

### CODE REVIEW

**Comment by dreispt** [comment]
Thanks for your contribution!

You should start by submitting your OCA CLA, and provide an expanded README better explaining the feature. I confess that I didn't understand the purpose it just by reading the code.

Then you'll find that TravisCI is your friend: check the build logs to see what needs fixing regarding teh OCA coding guidelines and best practices.

File: auth_oauth_multi_token/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'auth_oauth_multi_token',
+    'version': '1.0',
+    'license': 'AGPL-3',
```
Comment (gurneyalex) [inline auth_oauth_multi_token/__openerp__.py:7]: you include the text of the GPL license, but your module uses AGPL. You can probably drop the license text. If you use the recommended README.rst template for your module, you'll get a link to the license text.

File: auth_oauth_multi_token/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'auth_oauth_multi_token',
+    'version': '1.0',
+    'license': 'AGPL-3',
+    'author': "Florent de Labarre",
```
Comment (gurneyalex) [inline auth_oauth_multi_token/__openerp__.py]: please add 'Odoo Community Association (OCA)' to the list of authors.

File: auth_oauth_multi_token/__openerp__.py (hunk 1 above)
Comment (fmdl) [inline auth_oauth_multi_token/__openerp__.py:7]: corrected

**Comment by fmdl** [comment]
I'am already signed the CLA.

Regards

File: auth_oauth_multi_token/__openerp__.py (hunk 2 above)
Comment (fmdl) [inline auth_oauth_multi_token/__openerp__.py]: Added

**Comment by gurneyalex** [comment]
@IguanaYachts I'll check. It's probably a github login mismatch.

**Comment by gurneyalex** [comment]
CLA checked.

File: auth_oauth_multi_token/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# auth_oauth_multi_token
```
Comment (elicoidal) [inline auth_oauth_multi_token/README.md]: please remove this file

File: auth_oauth_multi_token/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    'summary': 'Multi token connection',
+    'category': 'Tool',
+    'website': 'www.iguana-yachts.com',
+    'depends': ['auth_oauth'],
+    'data': [
+        'views/res_users.xml',
+        'security/ir.model.access.csv',
+    ],
+    'demo': [],
```
Comment (elicoidal) [inline auth_oauth_multi_token/__openerp__.py]: remove the 2 empty keys

File: auth_oauth_multi_token/models/auth_oauth_multi_token.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline auth_oauth_multi_token/models/auth_oauth_multi_token.py:1]: @IguanaYachts   Thanks for your contribution to the OCA. 
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
- Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
- Please follow the OCA README template:
  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
- We recommend to include a screenshot as well to help new comers (nice to have)
- Please use simple headers in all py files following OCA templates
  https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3
- Please sort all imports in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

File: auth_oauth_multi_token/models/res_users.py [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+import openerp
+from openerp import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    oauth_access_token_ids = fields.One2many('auth.oauth.multi.token', 'user_id', 'Tokens', copy=False)
+    oauth_access_max_token = fields.Integer('Number of simultaneous connection', default=5, required=True)
```
Comment (elicoidal) [inline auth_oauth_multi_token/models/res_users.py]: s/connection/connections

**Review by susport — COMMENTED** [review]
Fix travis error, line too long in .py

File: auth_oauth_multi_token/views/res_users.xml [hunk 7]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (susport) [inline auth_oauth_multi_token/views/res_users.xml:1]: Change the xml label openerp and data to odoo in v9.0

**Comment by simahawk** [comment]
I took this over here https://github.com/OCA/server-tools/pull/1032

**Comment by lasley** [comment]
@simahawk - I edited your comment, you were linking to this issue instead of #1032

**Comment by simahawk** [comment]
@lasley thanks :sweat_smile:

**Pull request closed without merging**